app/src/main/resources/swagger/api-docs.json

```diff
@@ -30,6 +30,15 @@
             "type" : "string",
             "format" : "uuid"
           }
+        }, {
+          "name" : "namespace",
+          "in" : "query",
+          "description" : "Caller visibility context",
```

Comment:
i suggest to chang `namespace` description to something like this: `Tenant name`

Reply:
fixed with 46283983780a7ffd7b8559d985d1da95749e3bc2